docs/user-guide/install-overview.md

```diff
@@ -0,0 +1,17 @@
+# Overview
+
+The installation of Zowe&trade; consists of two independent processes: installing the Zowe server components either entirely on z/OS or a combination of z/OS and Docker, and then installing Zowe CLI on a desktop computer.
+
+The Zowe server components provide a web desktop that runs in a web browser providing a number of applications for z/OS users, together with an API mediation layer provides single-sign on (SSO), organization of the multiple zowe servers under a single website, and other capabilities useful for z/OS developers. 
+
+Because Zowe is a set of components, before installing Zowe, first determine which components you want to install and where you want to install them. This guide provides documentation for all of the components and it is split into sections so you can install as much as you need.
```

Comment:
```suggestion
Because Zowe is a set of components, before installing Zowe, use this guide to determine which components you want to install and where you want to install them.
```